node/node.go

```diff
@@ -725,6 +709,25 @@ func (n *Node) Dispatch() error {
 		n.Net.ManuallyTrack(bootstrapper.ID, bootstrapper.IP)
 	}
 
+	// Log a warning if we aren't able to connect to a sufficient portion of
+	// nodes.
+	go func() {
```

Comment:
I think that this would now incorrectly fire whenever `if requiredConns <= 0 {`.

Reply:
When running a 1-node local network this fired unexpectedly:
```
[06-28|15:19:45.577] WARN node/node.go:723 failed to connect to bootstrap nodes {"bootstrappers": "Validator Manager: (Size = 1)\n    Subnet[11111111111111111111111111111111LpoYY]: Validator Set: (Size = 0, Weight = 0)", "duration": "1m0s"}
```